sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -277,6 +277,28 @@ class ExplainSuite extends ExplainSuiteHelper with DisableAdaptiveExecutionSuite
     }
   }
 
+  test("SPARK-33850: explain formatted - check presence of subquery in case of AQE") {
+    withTable("df1") {
+      withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true") {
+        withTable("df1") {
+          spark.range(1, 100)
```

Comment:
I think its better to use temporary views in tests where possible.

Reply:
This is taken from another place. I'll open a followup PR and fix them.